test/test_spark_torch.py

```diff
@@ -343,14 +343,16 @@ def test_torch_direct_parquet_train(self):
                         store=store,
                         model=model,
                         optimizer=optimizer,
-                        loss=loss,
                         input_shapes=[[2]],
                         feature_cols=['features'],
                         label_cols=['y'],
                         batch_size=1,
                         epochs=3,
                         verbose=2)
 
+                    # To make sure that setLoss works with non-list loss.
```

Comment:
Looks like code paths are the same for constructor and this function now, right?  Was the issue before that users were calling `setLoss` which did not coerce the param into a list?

Reply:
I believe so. Passing the loss directly during instantiation works fine as we had it in the tests.

Reply:
I think we can move this back to the constructor, then.